### PR TITLE
Update sphere union spec

### PR DESCRIPTION
**Pull request by jglaser**
## Description

update the spec to be consistent with the spec for a single sphere (`radii` -> `diameters`)

## Motivation and Context

Resolves: N/A

## How Has This Been Tested?

N/A

doc build has been tested in #65 

## Change log

N/A

## Checklist:

- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/glotzerlab/gsd/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**GSD Contributor Agreement**](https://github.com/glotzerlab/gsd/blob/master/ContributorAgreement.md).
- [x] My name is on the [list of contributors](https://github.com/glotzerlab/gsd/blob/master/doc/credits.rst).

### CODE REVIEW

File: doc/shapes.rst [hunk 1]
```diff
@@ -143,20 +143,20 @@ Sphere unions
 
 Type: ``SphereUnion``
 
-A collection of spheres, defined by their radii and positions.
+A collection of spheres, defined by their diameters and positions.
```
Comment (bdice) [inline doc/shapes.rst]:
```suggestion
A collection of spheres, defined by their diameters and centers.
```

**Review by bdice — CHANGES_REQUESTED** [review]
@jglaser It looks like you committed the `fl.c` file. We don't include Cython generated files in the repo anymore since #57. I think it would be helpful to add a `.gitignore` entry for that file to prevent other developers from making the same mistake.

File: doc/shapes.rst (hunk 1 above)
Comment (lyrivera) [reply doc/shapes.rst]: Also, the header of the section should probably in caps (`Sphere Unions`), following the convention for he other shapes.

**Review by joaander — CHANGES_REQUESTED** [review]
As Bradley noted, please remove `fl.c` and add it to `.gitignore`

**Comment by vyasr** [comment]
You may also want to squash this PR when it's ready to remove the extraneous file from the git history.

**Review by bdice — APPROVED** [review]

**Review by joaander — APPROVED** [review]
Thanks!

**Pull request merged**